tests/cases/fourslash/toggleMultilineComment1.ts

```diff
@@ -0,0 +1,26 @@
+// Simple multiline comment and uncomment.
+
+//// let var1[| = 1;
+//// let var2 = 2;
+//// let var3 |]= 3;
+////
+//// let var4/* = 1;
+//// let var5 [||]= 2;

```

Comment:
It would just add multiline comment `/**/`. Btw, testing this I found an bug. Fixed and added tests for more edge cases. Check `toggleMultilineComment9.ts`.